static/app/components/group/externalIssuesList/streamlinedExternalIssueList.tsx

```diff
@@ -76,20 +80,8 @@ export function StreamlinedExternalIssueList({
     return <Placeholder height="25px" testId="issue-tracking-loading" />;
   }
 
-  const hasLinkedIssuesOrIntegrations = integrations.length || linkedIssues.length;
-  if (!hasLinkedIssuesOrIntegrations) {
-    return (
-      <AlertLink
-        type="muted"
-        to={`/settings/${organization.slug}/integrations/?category=issue%20tracking`}
-      >
-        {t('Track this issue in Jira, GitHub, etc.')}
-      </AlertLink>
-    );
-  }
-
   return (
-    <Fragment>
+    <Flex direction="row" wrap="wrap" gap={space(1)} flex={1}>
```

Comment:
side-question: should we refactor `Flex` to allow `gap=“sm”` and it actually reads the `sm` value from the theme? With `space()` being largely deprecated, this would be hard to achieve otherwise. Also, it would force everyone to use values from the spacing scale if the typings are: `gap: 'sm' | 'md' | ...` only